spec/v1/envelope.md

```diff
@@ -19,5 +19,48 @@ standard DSSE fields.
     specifying its schema.
 -   `payload` MUST be a base64-encoded JSON [Statement].
 
+## File naming convention
+
+If stored in a dedicated file by itself, and not as part of a [Bundle], an
+Envelope SHOULD use the suffix `.json`.
+
+-   For attestations intended for consumption by [in-toto-verify], an
+    Envelope containing an attestation about a particular SW supply chain
+    step `<step-name>` SHOULD be named `<step-name>.json`.
+-   For other verifiers, or cases in which a step name cannot be easily
+    determined, the attestation producer and consumer SHOULD agree on an
+    acceptable choice for filename: `<env-name>.json`.
```

Comment:
What's `env-name` and where does it come from?

Reply:
`env-name` is supposed to be an arbitrary envelope file name. I'm not a big fan of this choice, so if you have a suggestion for improvement, please let me know!

Reply:
Oh, that's not clear at all from the text, I'm afraid, which should explain this.

Reply:
Can update this.